app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -3358,7 +3358,7 @@ private int Enrol(int oFamilyId, int oInsureeId, int oPolicyId, int oPremiumId,
             }
 
             //get Policies
-            Query = "SELECT PolicyId, FamilyId, EnrollDate, StartDate, NULLIF(EffectiveDate,'null') EffectiveDate, ExpiryDate, Policystatus, PolicyValue, ProdId, OfficerId, PolicyStage, isOffline\n" +
+            Query = "SELECT PolicyId, ABS(FamilyId) AS FamilyId, EnrollDate, StartDate, NULLIF(EffectiveDate,'null') EffectiveDate, ExpiryDate, Policystatus, PolicyValue, ProdId, OfficerId, PolicyStage, isOffline\n" +
```

Comment:
Could you also check if PolicyId can be negative  ? if not can it be an issue to have twice the same ID (lack of constraints one form the database and one from the phone) 

Ideally we should have ABS for all IDs

Reply:
Yes. I will do it